test/test_add_page.py

```diff
@@ -29,3 +29,19 @@ def test_add_page_duration(tmp_path):
     pdf.add_page()
     pdf.cell(txt="Page 3")
     assert_pdf_equal(pdf, HERE / "add_page_duration.pdf", tmp_path)
+
+
+def test_break_or_add_page(tmp_path):
+    pdf = fpdf.FPDF()
+    pdf.set_auto_page_break(auto=True)
```

Comment:
As automated page break is enabled by default,
the only effect effect of this line is to reduce to zero the page bottom margin
that triggers page breaks.

Could you maybe specify `margin=0` as a second parameter,
to make the intention here a bit more explicit?